package.json

```diff
@@ -82,7 +83,7 @@
     "pinst": "^2.1.6",
     "prettier": "^2.2.1",
     "semantic-release": "^17.3.0",
-    "stylelint": "^14.0.1",
+    "stylelint": "^15.0.0",
     "stylelint-test-rule-tape": "^0.2.0",
```

Comment:
- [ ] [`stylelint-test-rule-tape`](https://github.com/stylelint/stylelint-test-rule-tape) is **archived**

Reply:
tape is replaced by [`jest-preset-stylelint`](https://github.com/stylelint/jest-preset-stylelint)

- https://stylelint.io/developer-guide/plugins/#testing
- https://stylelint.io/developer-guide/system-tests/